lib/Runtime/Library/ArrayBuffer.cpp

```diff
@@ -151,23 +151,43 @@ namespace Js
         }
     }
 
-    uint32 ArrayBuffer::GetByteLengthFromVar(ScriptContext* scriptContext, Var length)
+    uint32 ArrayBuffer::GetToIndex(Var value, int32 errorCode, ScriptContext *scriptContext, uint32 MaxAllowedLength, bool checkSameValueZero)
```

Comment:
No preference over here. The original spec ([http://www.ecma-international.org/ecma-262/6.0/](http://www.ecma-international.org/ecma-262/6.0/)) does not mention about the ToIndex. I can change that to ToIndex.

---
In reply to: [80845138](https://github.com/Microsoft/ChakraCore/pull/1629#discussion_r80845138) [](ancestors = 80845138)